satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -718,6 +730,12 @@ read all (
     where storagenode_storage_tally.interval_end_time >= ?
 )
 
+read all (
+    select storagenode_storage_tally
+    where storagenode_storage_tally.node_id = ?
+	where storagenode_storage_tally.interval_end_time >= ?
+)
+
```

Comment:
This query seems like it will be expensive. There's no apparent index for it, so I think the best it can do is use the node_id secondary index and scan through all of the records for that node.